src/components/drawer/drawer.tsx

```diff
@@ -5,20 +5,13 @@ import styles from './drawer.module.scss';
 
 export interface DrawerProps {
     className?: string;
-    initialIsOpen?: boolean;
     onClose?: () => void;
     title: string;
     children?: React.ReactNode;
 }
 
-export const Drawer = ({
-    className,
-    onClose: onclose,
-    title,
-    children,
-    initialIsOpen,
-}: DrawerProps) => {
-    const [isOpen, setIsOpen] = useState(initialIsOpen ?? false);
+export const Drawer = ({ className, onClose: onclose, title, children }: DrawerProps) => {
+    const [isOpen, setIsOpen] = useState(false);
 
     useEffect(() => {
         setIsOpen(true);
```

Comment:
?
so it us initially open?

Reply:
yes, the drawer component is written in a such strange way that is should only be rendered when it is supposed to be shown, then, it is rendered as closed and then useEffect takes place and sets internal isOpen to true. this is done like this in order to apply animation when it is opened.
but yes, we need to refactor this in future I think

Reply:
fixed drawer